File: GameLibrary/src/cc/lib/swing/AWTWrapLabel.java
package cc.lib.swing;

import javax.swing.JLabel;

@SuppressWarnings("serial")
public class AWTWrapLabel extends JLabel {

    public AWTWrapLabel() {
        super();
    }

    public AWTWrapLabel(String text) {
        setText(text);
    }
    

    @Override
    public void setText(String arg0) {
    	super.setText("<html>" + arg0.replace("\n", "</br>") + "</html>");
    }
    
    
    
}

File: SwingApps/src/cc/lib/swing/AWTWrapLabel.kt
package cc.lib.swing

import javax.swing.JLabel

class AWTWrapLabel : JLabel {
	constructor() : super() {}
	constructor(text: String) {
		setText(text)
	}

	override fun setText(arg0: String) {
		super.setText("<html>" + arg0.replace("\n", "</br>") + "</html>")
	}
}